pages/article/ArticleHandler.inc.php

```diff
@@ -71,9 +96,35 @@ function initialize($request, $args) {
 			$article = $articleDao->getById((int) $articleId, $journal->getId(), true);
```

Comment:
Hmmm... maybe to provide the submission revision here as parameter? 
This part of the code is executed when an editor previews an unpublished article, I believe. Maybe it is a good feature for previewing the unpublished versions?